pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_prerequisites.adoc

```diff
@@ -12,14 +12,16 @@
 
 = Prerequisites
 
+You must have the following environment to use {ProductName}. 
+
 .Operating Systems
 
 [NOTE]
 ====
-Operating system support affects the performance of the uploader more than other Pantheon components.
+Operating system support affects the performance of the uploader more than other {ProductName} components.
 ====
 
-Pantheon 2 is supported on the following operating systems:
+{ProductName} {ProductNumber} is supported on the following operating systems:
 
 * Fedora 30 or later
 * RHEL 7 CSB
```

Comment:
CSB is true, but it's specific to Red Hat. Maybe we should create ifdefs for RH/community?

Should change this to RHEL 7 CSB or later.